packages/flutter/test/widgets/binding_test.dart

```diff
@@ -67,18 +67,44 @@ void main() {
     WidgetsBinding.instance.addObserver(observer);
 
     setAppLifeCycleState(AppLifecycleState.paused);
-    expect(observer.lifecycleState, AppLifecycleState.paused);
+    expect(observer.accumulatedStates, <AppLifecycleState>[AppLifecycleState.paused]);
 
+    observer.accumulatedStates.clear();
     setAppLifeCycleState(AppLifecycleState.resumed);
-    expect(observer.lifecycleState, AppLifecycleState.resumed);
+    expect(observer.accumulatedStates,
+        <AppLifecycleState>[AppLifecycleState.hidden, AppLifecycleState.inactive, AppLifecycleState.resumed,]);
```

Comment:
nit: the trailing commas here and elsewhere are a bit strange.

Reply:
Fixed.